### PR TITLE
Adding getAllSessions

### PR DESCRIPTION
**Pull request by manoj9788**

### CODE REVIEW

File: appium/webdriver/webdriver.py [hunk 1]
```diff
@@ -701,6 +701,16 @@ def session(self):
         """
         return self.execute(Command.GET_SESSION)['value']
 
+    @property
+    def allSessions(self):
```
Comment (KazuCocoa) [inline appium/webdriver/webdriver.py]: Can you rename to snake case?

File: test/functional/ios/webdriver_tests.py [hunk 2]
```diff
@@ -24,9 +24,18 @@ class WebDriverTests(unittest.TestCase):
     def setUp(self):
         desired_caps = desired_capabilities.get_desired_capabilities('UICatalog.app.zip')
         self.driver = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
+        desired_caps = desired_capabilities.get_desired_capabilities('UICatalog.app.zip')
```
Comment (ki4070ma) [inline test/functional/ios/webdriver_tests.py]: This line is unnecessary.

File: test/unit/webdriver/webdriver_test.py [hunk 3]
```diff
@@ -257,6 +257,17 @@ def test_create_session_register_uridirect_no_direct_connect_path(self):
...
+    @httpretty.activate
+    def test_get_all_sessions(self):
+        driver = ios_w3c_driver()
+        httpretty.register_uri(
+            httpretty.GET,
+            appium_command('/sessions'),
+            body=json.dumps({'value': {'deviceName': 'iPhone Simulator', 'events': {'simStarted': [1234567891]}}})
+        )
+        session = driver.allSessions
+        assert len(session) != 0
```
Comment (ki4070ma) [inline test/unit/webdriver/webdriver_test.py]: `== 1` is better

File: test/functional/ios/webdriver_tests.py [hunk 4]
```diff
@@ -24,9 +24,18 @@ class WebDriverTests(unittest.TestCase):
...
+        desired_caps['deviceName'] = 'iPhone Xs Max'
+        desired_caps['wdaLocalPort'] = '8102'
+        self.driver1 = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
```
Comment (mykola-mokhnach) [inline test/functional/ios/webdriver_tests.py]: this is going to slow down all existing tests in that module. Can we move the init of the second driver into a separate test?

Comment (manoj9788) [reply test/functional/ios/webdriver_tests.py]: Changed.

File: test/unit/webdriver/webdriver_test.py (hunk 3 above)
Comment (manoj9788) [reply test/unit/webdriver/webdriver_test.py]: okay

File: appium/webdriver/webdriver.py (hunk 1 above)
Comment (manoj9788) [reply appium/webdriver/webdriver.py]: Sorry, yes. Coming from Java world :)

**Comment by KazuCocoa** [comment]
let me check ci

**Comment by KazuCocoa** [comment]
CI was happy by retrying it
Maybe, it was failed to create a new session because of something

**Comment by KazuCocoa** [comment]
will merge and clean a bit later.
Welcome to Appium world @manoj9788 :)

**Comment by manoj9788** [comment]
Thanks a lot @KazuCocoa

File: appium/webdriver/webdriver.py [hunk 5]
```diff
@@ -701,6 +701,16 @@ def session(self):
         """
         return self.execute(Command.GET_SESSION)['value']
 
+    @property
+    def all_sessions(self):
+        """ Retrieves all sessions that are open
+        Usage:
+            sessions = driver.allSessions
```
Comment (mykola-mokhnach) [inline appium/webdriver/webdriver.py]: the example is out of sync with function name

File: test/functional/ios/webdriver_tests.py [hunk 6]
```diff
@@ -24,10 +24,21 @@ class WebDriverTests(unittest.TestCase):
...
         self.driver.quit()
 
+    def testAllSessions(self):
+        desired_caps = desired_capabilities.get_desired_capabilities('UICatalog.app.zip')
+        desired_caps['deviceName'] = 'iPhone Xs Max'
+        desired_caps['wdaLocalPort'] = '8102'
+        self.driver1 = webdriver.Remote('http://localhost:4723/wd/hub', desired_caps)
+        self.assertEqual(2, len(self.driver.all_sessions))
+        if self.driver1:
+            self.driver1.quit()
```
Comment (mykola-mokhnach) [inline test/functional/ios/webdriver_tests.py]: this has to be in `finally` block

**Pull request merged**